scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController.java

```diff
@@ -122,4 +127,8 @@
 		return new ResponseEntity<>(null, null, HttpStatus.NO_CONTENT);
 	}
 
+	@ExceptionHandler(MissingServletRequestParameterException.class)
```

Comment:
Sorry for late reply, I missed your comment.
Scheduler has a ExceptionHandler class, you can put this method into ControllerExceptionHandler class.